nservicebus/messaging/routing-extensibility_content_core_[6,).partial.md

```diff
@@ -64,4 +64,6 @@ snippet:RoutingExtensibility-Distribution
 
 snippet:RoutingExtensibility-DistributionStrategy
 
+As of NServiceBus version 6.2 and later it is possible to override the virtual method `SelectDestination`. The method provides access to the `DistributionStrategyContext` that enables more advanced distribution scenarios like distributing based on the headers of the message. When `SelectDestination` is overridden, it is advised not to call `base.SelectDestination` since the base method calls into `SelectReceiver` for backward compatibility reasons. `SelectReceiver` can throw a `NotImplementedException` since it will never be called in this case.
```

Comment:
> The method provides access to the `DistributionStrategyContext` that enables more advanced distribution scenarios like distributing based on the headers of the message.

replace "like" with "such as". Possibly:

The method provides access to the `DistributionStrategyContext` enabling more advanced distribution scenarios such as message header based distribution.